tests/unit/stream_alert_rule_processor/test_parsers.py

```diff
@@ -90,6 +92,127 @@ class TestJSONParser(TestParser):
     def _parser_type(cls):
         return 'json'
 
+    def test_schema_fail(self):
+        """JSON Parser - Schema Match Failure"""
+        # setup
+        options = {}
+        schema = self.config['logs']['unit_test_simple_log']['schema']
+        data = json.dumps({'hey': 1})
+
+        # get parsed data
+        parsed_data = self.parser_helper(data=data, schema=schema, options=options)
```

Comment:
There's a few areas where `options` is instantiated in the test as an empty dict and just passed along. The `parser_helper` function does a `get` with the keyword args for `options`, so you can safely omit `options=...` from the helper call altogether. Please update here and other places below where options is not used.

Reply:
`parser_helper` doesn't do a `.get`, but I can make it do one

Reply:
sorry you're right it doesn't. you could also just pass `None` but either is fine